spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "GET #new" do
+    it "returns http success" do
+      get :new
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+end
```

Comment:
I used:
it { is_expected.to route(:get, "/users/new").to(controller: :users, action: :new) }
I will be interested to see if this is an acceptable format or whether I need the one you have used :)

Reply:
Elle, said the way I did the specs wasn't right for what I was doing, she pointed me back to the request specs we did in class

Reply:
Also, check Marta's PR for Elle's comments on where to put which type of specs for your controller